Tasks/MavenAuthenticateV0/package.json

```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "@types/node": "10.12.9",
     "@types/q": "^1.5.2",
-    "artifacts-common": "file:../../_build/Tasks/Common/artifacts-common-0.1.0.tgz",
+    "azure-pipelines-tasks-artifacts-common": "1.0.1",
```

Comment:
Did we already make a change to artifacts common?

Reply:
Yeah, I accidentally published a bad version the first time